easybuild/easyconfigs/j/JasPer/JasPer-2.0.14-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'CMakeMake'
+
+name = 'JasPer'
+version = '2.0.14'
+
+homepage = 'http://www.ece.uvic.ca/~frodo/jasper/'
```

Comment:
@akesandgren Please change to `https://`

Reply:
done